src/QMCWaveFunctions/EinsplineSet.cpp

```diff
@@ -15,7 +15,7 @@
 //////////////////////////////////////////////////////////////////////////////////////
 
 
-#include <Numerics/e2iphi.h>
+#include <CPU/e2iphi.h>
```

Comment:
"CPU/e2iphi.h"?

Reply:
In the past, <> is used for non-QMCPACK code. "" is used for QMCPACK code. We include header files based on a relative path to src.
In the spirit of organizing olders into CMake targets, a header file from a different top level folder is considered external.

QMCWaveFunctions/EinsplineSet.cpp belongs to qmcwfs target while 
Platforms/CPU/e2iphi.h belongs to platform_device target.
So the inclusion is considered external. "-Isrc/Platforms" is passed by target dependency.
That is why <> is used and there is no Platforms on the path.

Reply:
I suspect more cleanup is needed to satisfy this. No need to hold this up though.